packages/moxb/src/bind/BindImpl.ts

```diff
@@ -242,7 +254,7 @@ export class BindImpl<Options extends BindOptions<CustomData>, CustomData = unde
 
     get customData() {
         if (typeof this.impl.customData === 'function') {
-            return (this.impl.customData as Function)();
+            return (this.impl.customData as () => CustomData)();
         }
         if (this.impl.customData) {
             this.impl.customData;
```

Comment:
i think we are missing a return statement here (and a test)!

Reply:
Fixed the missing return. (But I haven't improved the poor test coverage.)